### PR TITLE
[CI][DOCKER] Update ci-gpu torch1.4 and onnx1.6

### PR DESCRIPTION
**Pull request by tqchen**
cc @masahi 

This is a docker image verified with testcases pass

### CODE REVIEW

File: docker/install/ubuntu_install_onnx.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ set -u
 set -o pipefail
 
 # fix to certain version for now
-pip3 install onnx==1.5.0
+pip3 install onnx==1.6.0
 pip3 install onnxruntime==1.0.0
 
 # torch depends on a number of other packages, but unhelpfully, does
```
Comment (masahi) [inline docker/install/ubuntu_install_onnx.sh]: have you updated torch?

Comment (tqchen) [reply docker/install/ubuntu_install_onnx.sh]: nice catch, the binary image is correct, somehow i missed the patch.
```
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
i>>> import torchvision
>>> torch.__version__
'1.4.0'
>>> torchvision.__version__
'0.5.0'
>>> 
```

**Comment by tqchen** [comment]
@masahi please take another look

File: docker/install/ubuntu_install_onnx.sh (hunk 1 above)
Comment (masahi) [reply docker/install/ubuntu_install_onnx.sh]: ok

**Pull request merged**